app/src/crypto.c

```diff
@@ -159,7 +174,7 @@ void crypto_extractPublicKeyHash(uint8_t *pubKeyHash, uint16_t pubKeyLen) {
     address_temp_t address_temp;
 
     cx_hash_sha256(publicKey, PK_LEN_SECP256K1, address_temp.hash_sha256, CX_SHA256_SIZE);
-    ripemd160(address_temp.hash_sha256, CX_SHA256_SIZE, pubKeyHash);         // RIPEMD-160
+    return ripemd160(address_temp.hash_sha256, CX_SHA256_SIZE, pubKeyHash);        // RIPEMD-160
```

Comment:
This function is called by `crypto_fillAddress_secp256k1`, but returned value is not checked.
As function does not throw exception anymore checking return value is mandatory.